voc_builder/interactive.py

```diff
@@ -360,9 +360,9 @@ def handle_cmd_no() -> NoActionResult:
 
     selector = ManuallySelector()
     if not ret.invalid_for_adding:
-        selector.discard_word(ret.word_sample)
+        selector.init_selector(ret.word_sample)
```

Comment:
The new method name is no better than the original. "Init" is a common word, putting a method starting with "init_" under an if condition is a bit confusing.